scripts/toolchain.lua

```diff
@@ -959,6 +965,32 @@ function toolchain(_buildDir, _libDir)
 			"NOMINMAX",
 		}
 
+	configuration { "GDKOne"}
+		targetdir (path.join(_buildDir, "gdkone/bin/$(Platform)/$(Configuration)"))
+		objdir (path.join(_buildDir, "gdkone/obj/$(Platform)/$(Configuration)"))
+		defines {
+			"_WIN64",
+			"NOMINMAX",
+			"UNICODE",
+			"_UNICODE",
+		}
+		buildoptions {
+		    "/favor:AMD64"
+		}
+
+	configuration { "GDKScarlett" }
+		targetdir (path.join(_buildDir, "gdkscarlett/bin/$(Platform)/$(Configuration)"))
+		objdir (path.join(_buildDir, "gdkscarlett/obj/$(Platform)/$(Configuration)"))
+		defines {
+			"_WIN64",
+			"NOMINMAX",
+			"UNICODE",
+			"_UNICODE",
+		}
+		buildoptions {
+		    "/favor:AMD64"
+		}
+
```

Comment:
Forgot to add:  removeflags { "StaticRuntime" }

Reply:
Add it, I didn't merge PR yet.